### PR TITLE
Fix transferfunction + new colormaps

### PR DESCRIPTION
**Pull request by moritz-h**

### CODE REVIEW

File: plugins/gui/src/TransferFunctionEditor.cpp [hunk 1]
```diff
@@ -144,7 +151,16 @@ std::array<std::tuple<std::string, PresetGenerator>, 12> PRESETS = {
     std::make_tuple("Cubehelix (default)", CubeHelixAdapter(0.5, -1.5, 1.0, 1.0)),
     std::make_tuple("Cubehelix (default, colorful)", CubeHelixAdapter(0.5, -1.5, 1.5, 1.0)),
     std::make_tuple("Cubehelix (default, de-pinked)", CubeHelixAdapter(0.5, -1.0, 1.0, 1.0)),
-    std::make_tuple("Cool-Warm (diverging)", ColormapAdapter<257>(CoolWarmColorMap))};
+    std::make_tuple("Cool-Warm (diverging)", ColormapAdapter<257>(CoolWarmColorMap)),
+    std::make_tuple("AccentMap (qualitative)", ColormapAdapter<8, true>(AccentMap)),
```
Comment (schulzch) [inline plugins/gui/src/TransferFunctionEditor.cpp]: This one is just called "Accent" and the amount of classes should be included, e.g. "5-class Accent".

File: plugins/gui/src/TransferFunctionEditor.cpp [hunk 2]
```diff
@@ -92,7 +93,12 @@ template <size_t PaletteSize> PresetGenerator ColormapAdapter(const float palett
             // Linear interpolation from palette.
             size_t i0 = static_cast<size_t>(std::floor(t * LastIndex));
             size_t i1 = static_cast<size_t>(std::ceil(t * LastIndex));
-            double it = std::fmod(t * LastIndex, LastIndex);
```
Comment (schulzch) [inline plugins/gui/src/TransferFunctionEditor.cpp:95]: Nice catch! 👍

File: plugins/gui/src/TransferFunctionEditor.cpp [hunk 3]
```diff
@@ -133,8 +139,9 @@ void RainbowAdapter(param::TransferFunctionParam::TFNodeType& nodes, size_t n) {
     }
 }
 
-std::array<std::tuple<std::string, PresetGenerator>, 12> PRESETS = {
-    std::make_tuple("Select...", [](auto& nodes, auto n) {}), std::make_tuple("Ramp", RampAdapter),
+std::array<std::tuple<std::string, PresetGenerator>, 20> PRESETS = {
```
Comment (schulzch) [inline plugins/gui/src/TransferFunctionEditor.cpp:142]: Now that we have more and more color maps, this tuple could/should be split into a struct, e.g..:
```
struct Preset {
	std::string name;
	enum class Scale {
		Sequential,
		Diverging,
		Qualitative
	} scale;
	std::string notice; // harmful, de-pinked, ...
	std::string reference; // ColorBrewer, Paper, ...
};
```
(just an idea)

**Review by schulzch — CHANGES_REQUESTED** [review]
Adding the amount of supported classes is important. Rest is optional

**Review by reinago — APPROVED** [review]

**Review by reinago — APPROVED** [review]

**Review by invor — APPROVED** [review]

**Review by schulzch — APPROVED** [review]

**Pull request merged**